index.js

```diff
@@ -31,6 +31,7 @@ function choo (opts) {
   start.router = router
   start.model = model
   start.start = start
+  start.stop = _store.stop
```

Comment:
hmmm, I feel we might want to add this to a plugin somehow - make it hard to access. does that make sense? - or is that a silly thought?